docker-compose.yml

```diff
@@ -16,6 +17,37 @@ services:
     command: --config /data/configdb/mongod.conf
     container_name: zkwasm-mongodb
     restart: always
+  prover-dry-run-service:
+    image: zkwasm:latest
+    runtime: nvidia
+    network_mode: "host"
+    user: "1001:1001"
+    build:
+      context: .
+      dockerfile: Dockerfile
+    volumes:
+      - ./prover_config.json:/home/zkwasm/prover-node-release/prover_config.json
+      - ./dry_run_config.json:/home/zkwasm/prover-node-release/dry_run_config.json
+      # Volume for the whole workspace directory to persist workspace data
+      # may have to consider handling changes in the workspace directory (delete volume probably)
+      - workspace-volume:/home/zkwasm/prover-node-release/workspace
+      # Volume for the logs directory to persist logs
+      - dry-run-logs-volume:/home/zkwasm/prover-node-release/logs/dry_run
+      # configure huge pages for the prover
+      - /dev/hugepages:/dev/hugepages
+    logging:
+      driver: "json-file"
+      options:
+        max-size: "10m"
+        max-file: "5"
+    environment:
+      - TZ=Etc/UTC
+    # Override default command to run as a dry-run service
+    command: bash -c "
+      time=$(date +%Y-%m-%d-%H-%M-%S)
```

Comment:
This not working, currently I can see the log in docker volume is :
prover_.log
dry_run_.log

time is not in the log file name